pkg/kubelet/rkt/rkt.go

```diff
@@ -1315,6 +1315,20 @@ func (r *Runtime) setupPodNetwork(pod *v1.Pod) (string, string, error) {
 	return netnsName, status.IP.String(), nil
 }
 
+// For a hostPath volume: rkt doesn't create any missing volume on the node/host so we need to create it
+func createHostPathVolumes(pod *v1.Pod) (err error) {
+	for _, v := range pod.Spec.Volumes {
+		if v.VolumeSource.HostPath != nil {
+			err = os.Mkdir(v.HostPath.Path, os.ModePerm)
```

Comment:
It won't actually end up as `0777` in the typical case because of umask.

e.g. my umask is 002 so when os.ModePerm applies it, you end up with 0775.